src/setup/node.rs

```diff
@@ -62,41 +49,15 @@ impl Node {
         // cleanup any previous runs (node.stop won't always be reached e.g. test panics, or SIGINT)
         self.cleanup()?;
 
-        // TODO: set initial peers/initial actions.
+        // generate config and start child process
         self.generate_config_file()?;
-
-        let (stdout, stderr) = match self.config.log_to_stdout {
-            true => (Stdio::inherit(), Stdio::inherit()),
-            false => (Stdio::null(), Stdio::null()),
-        };
-
-        let process = Command::new(&self.meta.start_command)
-            .current_dir(&self.meta.path)
-            .args(&self.meta.start_args)
-            .stdin(Stdio::null())
-            .stdout(stdout)
-            .stderr(stderr)
-            .spawn()
-            .expect("node failed to start");
-
-        self.wait_for_start().await;
+        let process = process::start(&self.meta, self.config.log_to_stdout);
```

Comment:
it might be a good idea for `start` to take `&self.config` as the second param, for future-proofing

Reply:
Done